src/components/Registration/Registration.jsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+import { RegistrationCards } from "./Registration.config";
+import styles from "./Registration.module.sass";
+import Link from "next/link";
+
+const Registration = () => {
+  return (
+    <>
+      <div className={styles.reg_header} key="reg-header">
```

Comment:
I think you don't need keys here
Check this topic:
https://www.developerway.com/posts/react-key-attribute